subworkflows/nf-core/fastq_align_bowtie2/main.nf

```diff
@@ -7,11 +7,11 @@ include { BAM_SORT_STATS_SAMTOOLS } from '../bam_sort_stats_samtools/main'
 
 workflow FASTQ_ALIGN_BOWTIE2 {
     take:
-    ch_reads          // channel: [ val(meta), [ reads ] ]
-    ch_index          // channel: /path/to/bowtie2/index/
-    save_unaligned    // val
-    sort_bam          // val
-    ch_fasta          // channel: /path/to/reference.fasta
+    ch_reads            // channel: [ val(meta), path(reads) ]
+    ch_index            // channel: [ path(index) ]     (/path/to/bowtie2/index/)
```

Comment:
Do we really need what's in the parentheses ?
If we want to standardize those comment, maybe taking this off would be better.
I also don't think some of them are really useful `(/path/to/reference.fasta)`